skeletons/bookmarklet/load_inspector.js

```diff
@@ -53,7 +53,7 @@
     if (e.data.name === 'version-mismatch') {
       injectEmberDebug('panes-' + e.data.version.replace(/\./g, '-') + '/ember_debug.js');
     }
-  });  
+  });
```

Comment:
Not sure what these characters are but they are causing some weirdness when deployed to S3.

<img width="122" alt="image" src="https://user-images.githubusercontent.com/3692/52094934-eed74f00-2575-11e9-923e-186baf71f2cf.png">